compiler/rustc_session/src/session.rs

```diff
@@ -1354,6 +1357,7 @@ pub fn build_session(
         target_features: FxHashSet::default(),
         known_attrs: Lock::new(MarkedAttrs::new()),
         used_attrs: Lock::new(MarkedAttrs::new()),
+        if_let_suggestions: Lock::new(Default::default()),
```

Comment:
```suggestion
        if_let_suggestions: Default::default(),
```

Reply:
Should I do this change for all the other cases above?

Reply:
Done (only for this one line).